tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3611,4 +3611,37 @@ sdg/SG_GEN_PARL.SEX--F,proportion of seats held by women in national parliaments
 Count_Person_Upto18Years,children population count
 Count_Person_18OrMoreYears,adult population count
 Count_Person_65OrMoreYears,senior population count
-dc/x6l3mg60hpe7c,"Consumer Price Index of Consumer Goods And Services (Seasonally Unadjusted): Urban Consumer; Consumer Price Index for All Urban Consumers"
\ No newline at end of file
+dc/x6l3mg60hpe7c,"Consumer Price Index of Consumer Goods And Services (Seasonally Unadjusted): Urban Consumer; Consumer Price Index for All Urban Consumers"
+Count_Person_15OrMoreYears_EducationSector_Employed,Number of people employed in education sector
+Count_Person_15OrMoreYears_Manufacturing_Employed,Number of people employed in Manufacturing
+Count_Person_15OrMoreYears_SpecializedScientificAndTechnicalActivities_Employed,Number of people employed in specialized scientific or technical activities
+Count_Person_15OrMoreYears_AgricultureHuntingForestryCuttingTrees_Employed,Number of people employed in agriculture, forestry or fishing
+Count_Person_15OrMoreYears_HumanHealthAndSocialWorkActivities_Employed,Number of people employed in social work activities
+Count_Person_15OrMoreYears_PublicAdministrationAndDefence_Employed,Number of people employed in public administration or defence
+Count_Person_15OrMoreYears_InformationAndTelecommunication_Employed,Number of people employed in Information and communication
+Count_MortalityEvent_Infant_Rural,Rural infant mortality rate

```

Comment:
is this stat var supposed to be the rate or the count? From the dcid, it looks like it's just the count?